### PR TITLE
Test7

### PR DESCRIPTION
**Pull request by dikshatest**
This branch is ready for the code review @dikshatest2

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Hi
 # Hello
+assume this a block of code needed 
 # New branch 
 # Diksha
 This is just the testing
+Changing
+bla bla bla
```
Comment (dikshatest2) [inline readme.md]: Please remove one bla from the bla bla bla

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Hi
 # Hello
+assume this a block of code needed
```
Comment (dikshatest2) [inline readme.md]: Start the assume with cap A

**Review by dikshatest2 — CHANGES_REQUESTED** [review]
these fields needs changing

File: readme.md (hunk 2 above)
Comment (dikshatest) [reply readme.md]: Done

**Comment by dikshatest** [comment]
@dikshatest2 can you review this again

**Review by dikshatest2 — APPROVED** [review]

**Comment by dikshatest2** [comment]
@bill this case is ready fot s.up

**Pull request merged**